Source/MonoGame.Extended/FPSCounter.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Xna.Framework;
+
+namespace MonoGame.Extended
+{
+    public static class FPSCounter
+    {
+        #region Private Fields
```

Comment:
Please don't use regions. They just add to the clutter. If the class is small enough, there's absolutely no benefit. If the class is big, it probably should be broken up into smaller classes